proposals/continuations/Overview.md

```diff
@@ -41,11 +41,16 @@ Based on [typed reference proposal](https://github.com/WebAssembly/function-refe
     - and `($ct' = cont $ft')*`
     - and `([te2*] -> [t2*] <: $ft')*`
```

Comment:
To address Frank's concern:
```suggestion
    - and `$ft' = [t1'*] -> [t2'*]`
    - and `([te2*] -> [t2*] <: [t1'*] -> [t2'*])*`
```
(Subtyping on raw function types is defined structurally, so this works.)